.github/file-paths.yaml

```diff
@@ -130,3 +130,8 @@ visualizations:
   - "frontend/src/metabase/visualizations/**"
   - "frontend/src/metabase/static-viz/**"
   - "src/metabase/pulse/**"
+
+cljs:
+  - *shared_sources
+  - *shared_specs
+  - ".github/workflows/cljs.yml"
```

Comment:
👌

Reply:
@bshepherdson I think we should include https://github.com/metabase/metabase/blob/master/shadow-cljs.edn in here somewhere.

Sources? Or directly in `cljs`?
Once that's solved, this PR looks good to go 🚀

Reply:
Yeah, my reasoning was that one might alter the output path in that file, for example. And this workflow wouldn't run, and wouldn't give you feedback that it's broken. At least not until the first next time it runs.